scripts/test-runner.js

```diff
@@ -224,14 +219,16 @@ process.on('SIGINT', shutdownProcesses);
 process.on('SIGTERM', shutdownProcesses);
 process.on('SIGHUP', shutdownProcesses);
 
-startRC(false).then(() => {
+startRC(!runTests).then(() => {
```

Comment:
Start rc in background if tests won't run. This way it can survive after the program exists. (via nohup and start /cmd)